sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/BatchSpannerRead.java

```diff
@@ -138,6 +138,9 @@ public void processElement(ProcessContext c) throws Exception {
       BatchReadOnlyTransaction batchTx =
           spannerAccessor.getBatchClient().batchReadOnlyTransaction(tx.transactionId());
       ReadOperation op = c.element();
+      boolean dataBoostEnabled =
+          config.getDataBoostEnabled() != null
+              && Boolean.TRUE.equals(config.getDataBoostEnabled().get());
```

Comment:
There is no need for the `Boolean.TRUE.equals()`

Reply:
Since config.getDataBoostEnabled().get() is a Boolean, I thought it can be null as well. Thats why I wrote this way. Can it never be null?